kong/plugins/aws-lambda/handler.lua

```diff
@@ -94,12 +95,14 @@ local function extract_proxy_response(content)
 
   local headers = serialized_content.headers or {}
   local body = serialized_content.body or ""
+  local isBase64Encoded = serialized_content.isBase64Encoded or false
   headers["Content-Length"] = #body
 
   return {
     status_code = tonumber(serialized_content.statusCode),
     body = body,
     headers = headers,
+    isBase64Encoded = isBase64Encoded
```

Comment:
instead of returning this flag, shouldn't we just decode here and not return the flag? func is called `extract_proxy_response` so that would make sense no?

Reply:
Yes, that makes sense. I will make that change.